lib/gollum-lib/page.rb

```diff
@@ -240,7 +238,7 @@ def format
     #
     # Returns a Gollum::Markup instance.
     def markup
-      @markup ||= @wiki.markup_classes[format].new(self)
```

Comment:
@dometto I've skimmed the PR a little bit, and I have one question. How the `format` thing is going to be applied to this new approach? I mean in the old approach we got the proper `Markup` class based on the format, now we get the generic class `Gollum::Markup`. I supposed that the class will act accordingly to the page format?

Reply:
Nice then 👍. Everything looks cleaner now. Good work!